Uplift/Views/GymDetail/GymDetailWeekView.swift

```diff
@@ -14,31 +14,33 @@ import SnapKit
 class GymDetailWeekView: UIView {
 
     // MARK: - Public
-    var selectedDay: WeekDay = .sunday
-    weak var delegate: WeekDelegate? {
-        didSet {
-            self.delegate?.didChangeDay(day: selectedDay)
-        }
-    }
+    var selectedDay: WeekDay
+    var selectedDayIndex: Int
+    weak var delegate: WeekDelegate?
 
     // MARK: - Display
     private var weekdayCollectionView: UICollectionView!
     private let daysSize = CGSize(width: 24, height: 24)
     private let interitemSpace: CGFloat = 19
 
     // MARK: - Info
-    private var days: [WeekDay] = [.monday, .tuesday, .wednesday, .thursday, .friday, .saturday, .sunday]
+    private var days: [WeekDay] = [.sunday, .monday, .tuesday, .wednesday, .thursday, .friday, .saturday]
+    // [.monday, .tuesday, .wednesday, .thursday, .friday, .saturday, .sunday]
```

Comment:
also still waiting to hear back from design r.e. whether the calendar should start with sunday or monday